src/auth/authGuard.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
What is the essence of this at the moment? Do you need a Role or the naming of the class doesn't match the implementation?

Reply:
okay boss